### PR TITLE
Test

### PR DESCRIPTION
**Pull request by rbuzatu90**
test

### CODE REVIEW

File: Calculul cheltuielilor salariale lunare/calcul_salar.py [hunk 1]
```diff
@@ -32,6 +32,18 @@ def ded(persoane):
         elif salar_brut >= 3000:
             deducere = 0
             return deducere
+        elif salar_brut >= 1501 and salar_brut < 3000:
+            if persoane == 0:
+                deducere = 300*(1-((salar_brut - 1500)/1500.))
+            elif persoane == 1:
+                deducere = 400*(1-((salar_brut - 1500)/1500.))
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/calcul_salar.py]: anabla bala

File: Calculul cheltuielilor salariale lunare/calcul_salar.py [hunk 2]
```diff
@@ -48,10 +60,13 @@ def ded(persoane):
 
     sanatate_angajator = salar_brut * 5.2/100
     Somaj_angajator = salar_brut * 0.5/100
+    Fond_creantesalariale = salar_brut * 0.25/100
+    Cas_angajator = salar_brut * 15.8/100
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/calcul_salar.py]: ana maria 2

File: Calculul cheltuielilor salariale lunare/calcul_salar.py (hunk 1 above)
Comment (rbuzatu90) [reply Calculul cheltuielilor salariale lunare/calcul_salar.py]: Taci ca nu-i bun

File: Calculul cheltuielilor salariale lunare/Date.csv [hunk 3]
```diff
@@ -1,5 +1,7 @@
 65190,Ianuarie
 73677,Februarie
+732177,Februarie
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/Date.csv]: test1

File: Calculul cheltuielilor salariale lunare/calcul_salar.py [hunk 4]
```diff
@@ -48,10 +60,13 @@ def ded(persoane):
...
     sanatate_angajator = salar_brut * 5.2/100
     Somaj_angajator = salar_brut * 0.5/100
+    Fond_creantesalariale = salar_brut * 0.25/100
+    Cas_angajator = salar_brut * 15.8/100
     Cas_angajator = salar_brut * 15.8/100
     cci = salar_brut * 0.85/100
     Risc_accidente = salar_brut * 0.4/100
-    Fond_creantesalariale = salar_brut * 0.25/100
+    cci = salar_brut * 0.85/100
+    Risc_accidente = salar_brut * 0.4/100
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/calcul_salar.py]: test2

File: Calculul cheltuielilor salariale lunare/run_this.py [hunk 5]
```diff
@@ -19,6 +19,7 @@ def numar_a():
     global numar_angajati
     a =  len(file) #numar angajati incarcati in fisier
     numar_angajati = a
+    print numar_angajati
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/run_this.py]: test3

File: Calculul cheltuielilor salariale lunare/calcul_salar.py [hunk 6]
```diff
@@ -32,6 +32,18 @@ def ded(persoane):
...
+        elif salar_brut >= 1501 and salar_brut < 3000:
+            if persoane == 0:
+                deducere = 300*(1-((salar_brut - 1500)/1500.))
+            elif persoane == 1:
+                deducere = 400*(1-((salar_brut - 1500)/1500.))
+            elif persoane == 2:
+                deducere = 500*(1-((salar_brut - 1500)/1500.))
+            elif persoane == 3:
+                deducere = 600*(1-((salar_brut - 1500)/1500.))
+            else:
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/calcul_salar.py]: 1

File: Calculul cheltuielilor salariale lunare/run_this.py [hunk 7]
```diff
@@ -559,4 +561,4 @@ def sterge_angajat():
 
 calcul_salar.raise_frame(f1)
 
-root.mainloop()
\ No newline at end of file
+root.mainloop()
```
Comment (ampintilie) [inline Calculul cheltuielilor salariale lunare/run_this.py]: 2

**Pull request closed without merging**